common/next/next.config.js

```diff
@@ -75,11 +78,43 @@ const createConfig =
         return config;
       },
       eslint: {
+        ...validDefaultConfig.eslint,
         ignoreDuringBuilds: !options.lintBuilds,
       },
       transpilePackages: ['@weco/common'],
+      experimental: {
+        ...validDefaultConfig.experimental,
+        mdxRs: true,
+        outputFileTracingRoot: path.join(__dirname, '../../'),
+      },
+      reactStrictMode: true,
     });
     return nextConfig;
   };
 
+// TODO: check the build output without this function whenever next has an update
+/**
+ * this will only be neccessary until the default config is updated OR the config validator is updated.
+ * how I check this? I don't know besides doing it manually
+ */
+const cleanInvalidValues = defaultConfig => {
+  const config = { ...defaultConfig };
+
+  const invalidProperties = [
+    'webpackDevMiddleware',
+    'configOrigin',
+    'target',
+    'assetPrefix',
+    'i18n',
+  ];
+  for (let index = 0; index < invalidProperties.length; index++) {
+    const property = invalidProperties[index];
+    delete config[property];
+  }
+
+  config.amp.canonicalBase = '.';
```

Comment:
Could we consider this one part of the invalid properties for now?

Reply:
it's gone now